### PR TITLE
fix(markdown): format code blocks

### PR DESCRIPTION
**Pull request by nimbinatus**
Patch up the configuration info to display the code blocks properly.

### CODE REVIEW

File: themes/default/layouts/shortcodes/configure-gcp.html [hunk 1]
```diff
@@ -3,5 +3,5 @@
 `config` command using the project's ID: ```bash gcloud config set project
 <YOUR_GCP_PROJECT_ID>
     ``` Next, Pulumi requires default application credentials to interact with your Google Cloud resources, so run `auth application-default login` command to obtain those
-    credentials. ```bash gcloud auth application-default login ```</YOUR_GCP_PROJECT_ID
+    credentials: ```bash $ gcloud auth application-default login ```</YOUR_GCP_PROJECT_ID
```
Comment (susanev) [inline themes/default/layouts/shortcodes/configure-gcp.html:6]:
```suggestion
    credentials: ```bash $ gcloud auth application-default login ```
```

File: themes/default/layouts/shortcodes/configure-gcp.html [hunk 2]
```diff
@@ -3,5 +3,5 @@
...
 >
```
Comment (susanev) [inline themes/default/layouts/shortcodes/configure-gcp.html:7]:
```suggestion

```

**Review by susanev — APPROVED** [review]

**Comment by susanev** [comment]
alright @nimbinatus i didnt realize this as an html file and not a markdown file, I've updated it with what you want, at least i think. pls take another look.

File: themes/default/layouts/shortcodes/configure-gcp.html [hunk 3]
```diff
@@ -1,7 +1,27 @@
...
 account](https://cloud.google.com/sdk/docs/authorizing#authorizing_with_a_user_account). If `gcloud` is not configured to interact with your Google Cloud project, set it with the
-`config` command using the project's ID: ```bash gcloud config set project
-<YOUR_GCP_PROJECT_ID>
-    ``` Next, Pulumi requires default application credentials to interact with your Google Cloud resources, so run `auth application-default login` command to obtain those
-    credentials. ```bash gcloud auth application-default login ```</YOUR_GCP_PROJECT_ID
->
+`config` command using the project's ID:
+
+<div class="highlight">
+    <pre class="chroma"><code class="language-bash" data-lang="bash"><span class="no-select">$ </span> gcloud config set project your-gcp-project-id</code></pre>
```
Comment (susanev) [inline themes/default/layouts/shortcodes/configure-gcp.html:6]:
```suggestion
    <pre class="chroma"><code class="language-bash" data-lang="bash"><span class="no-select">$ </span>gcloud config set project your-gcp-project-id</code></pre>
```

**Comment by guineveresaenger** [comment]
Partial fix for https://github.com/pulumi/home/issues/2270, by the way ❤️

**Review by guineveresaenger — APPROVED** [review]
If there were multiple Approve buttons, I would smash all of them!

**Comment by nimbinatus** [comment]
I'm not quite sure what happened with my branch as that first commit isn't what I have locally. The main issue is fixed, though, so it's readable now (though I'd rather make it match every other getting started guide we have to use the angle brackets for the variable, I'm not going to worry about it). Thank you!

**Comment by susanev** [comment]
i think we can escape the angle brackets in the html to prevent the linter from insisting that it needs to be closed, i can do a follow-up pr

**Pull request merged**